README.md

```diff
@@ -1,15 +1,27 @@
-<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
+<img src="https://opensearch.org/wp-content/uploads/2025/01/opensearch_logo_default.svg" height="64px">
 
 - [Observability](#observability)
-- [Code Summary](#code-summary)
-- [Plugin Components](#plugin-components)
-- [Documentation](#documentation)
-- [Contributing](#contributing)
-- [Getting Help](#getting-help)
-- [Code of Conduct](#code-of-conduct)
-- [Security](#security)
-- [License](#license)
-- [Copyright](#copyright)
+  - [Code Summary                                                                                      |](#code-summary--------------------------------------------------------------------------------------)
```

Comment:
Why does the formatting have so much white space on this line?

Reply:
The previous version was outdated and this is auto corrected by the markdown reader

Reply:
and here is why: https://github.com/opensearch-project/dashboards-observability/pull/2413/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L18


seems like we made this mistake a while ago. I have also corrected this

Reply:
Thanks for the fix!